### PR TITLE
[only for test] "[fix](ubsan) fix some Undefined behavior #35393"

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

File: be/src/util/frame_of_reference_coding.h [hunk 1]
```diff
@@ -34,9 +34,13 @@ inline uint8_t bits_less_than_64(const uint64_t v) {
...
 // See https://stackoverflow.com/questions/28423405/counting-the-number-of-leading-zeros-in-a-128-bit-integer
 inline uint8_t bits_may_more_than_64(const uint128_t v) {
+    // See https://stackoverflow.com/questions/49580083/builtin-clz-returns-incorrect-value-for-input-zero
+    if (v == 0) {
+        return 0;
+    }
     uint64_t hi = v >> 64;
     uint64_t lo = v;
-    int z[3] = {__builtin_clzll(hi), __builtin_clzll(lo) + 64, 128};
+    int z[3] = {hi == 0 ? 64 : __builtin_clzll(hi), (lo == 0 ? 64 : __builtin_clzll(lo)) + 64, 128};
```
Comment (HappenLee) [inline be/src/util/frame_of_reference_coding.h:43]: should be an inline function __builtin_clzll

File: be/src/vec/columns/column_nullable.cpp [hunk 2]
```diff
@@ -338,7 +338,7 @@ void ColumnNullable::insert_from(const IColumn& src, size_t n) {
     const auto& src_concrete = assert_cast<const ColumnNullable&>(src);
     get_nested_column().insert_from(src_concrete.get_nested_column(), n);
     auto is_null = src_concrete.get_null_map_data()[n];
-    _has_null |= is_null;
```
Comment (HappenLee) [inline be/src/vec/columns/column_nullable.cpp:341]: init `_has_null`

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run P0

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Pull request closed without merging**